src/guide/list.md

```diff
@@ -268,25 +268,23 @@ Similar to template `v-if`, you can also use a `<template>` tag with `v-for` to
 Note that it's **not** recommended to use `v-if` and `v-for` together. Refer to [style guide](../style-guide/#avoid-v-if-with-v-for-essential) for details.
 :::
 
-When they exist on the same node, `v-for` has a higher priority than `v-if`. That means the `v-if` will be run on each iteration of the loop separately. This can be useful when you want to render nodes for only _some_ items, like below:
+When they exist on the same node, `v-if` has a higher priority than `v-for`. That means that if `v-if` condition depends on the property from `v-for` scope, we will have an error: 
 
 ```html
 <li v-for="todo in todos" v-if="!todo.isComplete">
+<!-- Property "todo" was accessed during render but is not defined on instance. -->
```

Comment:
I think this needs moving up a line. It's potentially confusing because the next line also mentions `todo`. Perhaps that line should be removed?

Reply:
I don't think we should remove this line altogether because we need to show that this code snippet is incorrect. I changed a message to reflect this better

Reply:
Ah, sorry, I wasn't clear.

I wasn't suggesting removing the comment line. I was suggesting removing the next line, `{{ todo }}`. The problem is that it mentions `todo`, so it wasn't entirely clear whether the error message was referring to the `v-if` or the `{{ todo }}`.

Moving the comment up has helped. I just wonder whether replacing the `{{ todo }}` with, e.g. `...` or a comment would remove any potential for someone to misunderstand? One of the earlier examples uses `<!-- content -->`, so perhaps that?

```html
<li v-for="todo in todos" v-if="!todo.isComplete">
  <!-- content -->
</li>
```

Now I look a bit closer there's actually another problem with the `{{ todo }}`. The `v-if="todo.isComplete"` implies that `todo` is an object with an `isComplete` property but the `{{ todo }}` seems to be treating `todo` like it's a string. It isn't technically wrong, I just worry about someone getting distracted by the `{{ todo }}` and thinking it has something to do with the error.

If you do decide to change it then the next example would need updating to match.